pages/about.html

```diff
@@ -0,0 +1,132 @@
+                    </ul>
+                </nav>
+            </div>
+        </section>
+        <section class="mobile-menu">
+            <i class="fa fa-close" style="color:#ffffff;"></i>
+            <ul>
+                <li><a href="../index.html" class="tag">Home</a></li>
+                <li><a href="../index.html">Program</a></li>
+                <li><a href="../index.html">Join</a></li>
+                <li><a href="../index.html">Sponsor</a></li>
+                <li><a href="../index.html">News</a></li>
+                <li><a href="../index.html" class="sponsor tag">Sponsor</a></li>
+            </ul>
+        </section>
+    </header>
+
+    <main>
+        <section class="hero-section">
+            <section class="hero-article">
+                <h1>EmpowerHer Summit 2023:</h1>
+                <h2>Building the level ground for growth</h2>
+                <hr class="pinkhr">
+                <p>EmpowerHer Foundation is a non-profit organization dedicated to empowering women and
+                    fostering their personal and professional growth. We believe
+                    that when women are equipped with knowledge, skills, and support,
+                    they can overcome barriers, achieve their goals, and make a positive
+                    impact in their lives and communities.
+                </p>
+                <p>Feel free to contact us via Email for any further
+                    questions about EmpowerHer summit 2023!
+                    <br><a href="mailto: uzochukwuwinnie@gmail.com">empowerher@foundation.org</a>
+                </p>
+            </section>
+        </section>
+
+        <section class="about-foundation">
+            <h2>About EmpowerHer</h2>
+            <hr class="pink-hr">
+            <p>Since our inception in 2021, we have been on a mission to provide
+                comprehensive programs and services that uplift and empower
+                women from all walks of life. Through our diverse range of
+                initiatives, we have had the privilege of supporting and
+                positively impacting the lives of over 300 women.
+            </p>
+            <img src="../assets/images/Color logo with background.png" alt="empowerher logo">
+        </section>
+        <hr>
+        <section class="past-summits">
+            <h2>EmpowerHer Past Summits</h2>
+            <hr class="pink-hr">
+            <p>We send our attendees back home feeling renewed and energized!</p>
+            <section>
+                <div class="past-photo recent">
+                    <h3>2022</h3>
+                    <img src="../assets/images/2022.jpg" alt="women from different ethnicity standing in a group photo">
+                </div>
+                <div class="past-photo older">
+                    <h3>2021</h3>
+                    <img src="../assets/images/2021.jpg" alt="five black women sitting and smiling at the camera">
+                </div>
+            </section>
```

Comment:
- [ ] It would be great to reduce to the size of your images as you may notice how big they are.
- [ ] It would also be great to readjust the size of your sponsors' logo to make sure your page has a great touch and feel as is the case in the design.
### Your design :point_down: 
![image](https://github.com/Wineshuga/EmpowerHER-Foundation/assets/97100091/67dea531-a114-42dc-bb34-16c7c584f00e)
![image](https://github.com/Wineshuga/EmpowerHER-Foundation/assets/97100091/8bb66e4e-7ec7-4f6f-9507-d355e6082f05)
### Original design :point_down: 
![image](https://github.com/Wineshuga/EmpowerHER-Foundation/assets/97100091/498bec73-fd66-4caa-a36e-08d7cce95e2e)

![image](https://github.com/Wineshuga/EmpowerHER-Foundation/assets/97100091/369df1ce-d614-4f68-a4c8-052c15a9503f)